src/components/YourCompany/NameTranslations/ListNameTranslation.vue

```diff
@@ -207,4 +210,11 @@ export default class ListNameTranslation extends Vue {
     min-height: 0;
     padding: 0 1rem 0 0.5rem;
   }
+  .v-list-item__subtitle {
+  color: var(--v-primary-base) !important;
```

Comment:
Not entirely sure, but this could be the same as `$app-blue` which we have defined in the `theme.scss`.
For consistency, i would use our colour definitions from the theme, just in the event that we have overridden something from the vuetifys base theme.

Reply:
I copied this from ListPeopleAndRoles.Vue... to make the remove link blue. I can try $app-blue :)

Reply:
Changed the color as per advice :)